tests/test-config.sh

```diff
@@ -53,3 +53,20 @@ assert_file_has_content repo/config "Nightly Flathub"
 assert_file_has_content repo/config "false"
 assert_file_has_content repo/config "http://example.com/ostree/"
 echo "ok config set"
+
+${CMD_PREFIX} ostree config --repo=repo set core.lock-timeout-secs 60
+assert_file_has_content repo/config "lock-timeout-secs=60"
+${CMD_PREFIX} ostree config --repo=repo unset core.lock-timeout-secs
+assert_not_file_has_content repo/config "lock-timeout-secs=60"
```

Comment:
A stronger check here would be that the key doesn't exist at all (i.e. just `lock-timeout-secs=`). And maybe corroborate that with sanity checking that `ostree config get` erroring out?

Reply:
You're right; I made both changes and force-pushed the branch